velox/docs/functions/spark/math.rst

```diff
@@ -262,3 +262,16 @@ Mathematical Functions
         SELECT unhex("b2323"); -- \x0B##
         SELECT unhex("G"); -- NULL
         SELECT unhex("G23"); -- NULL
+
+.. spark:function:: width_bucket(x, bound1, bound2, n) -> bigint
+
+    Returns the bucket number to which ``x`` would be assigned in an equiwidth histogram with ``n`` buckets,
+    in the range ``bound1`` to ``bound2``.
+    `n` must be greater than zero and be less than Long.MaxValue, `x` cannot be NaN, `bound1` bound cannot equal `bound2`,
```

Comment:
Let's clarify types of x, bound1 and bound2. I assume these are DOUBLE. Let's also clarify whether NaN, positive or negative infinity are allowed and whether bound1 must be strictly less than bound2.

Reply:
done